modules/account/src/main/kotlin/edu/stanford/spezi/module/account/manager/UserSessionManager.kt

```diff
@@ -32,19 +32,20 @@ internal class UserSessionManagerImpl @Inject constructor(
 ) : UserSessionManager {
     private val logger by speziLogger()
 
-    override suspend fun uploadConsentPdf(pdfBytes: ByteArray): Result<Unit> = withContext(ioDispatcher) {
-        runCatching {
-            val currentUser = firebaseAuth.currentUser ?: error("User not available")
-            val inputStream = ByteArrayInputStream(pdfBytes)
-            logger.i { "Uploading file to Firebase Storage" }
-            val uploaded = firebaseStorage
-                .getReference("users/${currentUser.uid}/signature.pdf")
-                .putStream(inputStream)
-                .await().task.isSuccessful
+    override suspend fun uploadConsentPdf(pdfBytes: ByteArray): Result<Unit> =
+        withContext(ioDispatcher) {
+            runCatching {
+                val currentUser = firebaseAuth.currentUser ?: error("User not available")
+                val inputStream = ByteArrayInputStream(pdfBytes)
+                logger.i { "Uploading file to Firebase Storage" }
+                val uploaded = firebaseStorage
+                    .getReference("patients/${currentUser.uid}/consent/signature.pdf")
```

Comment:
On iOS we currently store it under `consent/\(dateString).pdf` ("yyyy-MM-dd_HHmmss" as the date string format): https://github.com/StanfordBDHG/ENGAGE-HF-iOS/blob/main/ENGAGEHF/ENGAGEHFStandard.swift#L193. 

We should use the same logic here as well for now. We might want to adjust this in the future.

Reply:
We are currently checking whether the file has already been signed. If we make the change to the date naming, we will no longer know the name of the file at a later date and could only check whether there is any file in the folder. The date feels redundant because we save the creation and update date of the file anyway.  This could also lead to problems if we later have several consent files. However, if this is wanted, then I can adjust it @PSchmiedmayer  :)

Reply:
To recap the discussion in the meeting today: We will use a `/patients/{PATIENT_ID}/consent.pdf` path without any version numbers nor a lot of nesting for now. This needs to be aligned with the iOS app @nriedman.